src/mesh_element/Element_Definition.cc

```diff
@@ -105,6 +108,11 @@ Element_Definition::Element_Definition(
   }
   Require(side_type_.size() == number_of_sides_);
   for (unsigned i = 0; i < number_of_sides_; ++i) {
+    std::cout << " side number " << i << " is side type " << side_type_[i]
+              << std::endl;
+    std::cout << "    which is " << elem_defs_[side_type_[i]].get_name()
+              << " element " << std::endl;
+
```

Comment:
Do these print statements belong in the final version? Are these debug statements that should be removed?